page 21180
/bbs/Gossiping/M.1495711693.A.FDB.html download to ptt/Gossiping/21180_0.txt, time: 10.500000
/bbs/Gossiping/M.1495711732.A.94A.html download to ptt/Gossiping/21180_1.txt, time: 4.660000
/bbs/Gossiping/M.1495711734.A.24E.html download to ptt/Gossiping/21180_2.txt, time: 3.890000
empty download to ptt/Gossiping/21180_3.txt, time: 0.000000
/bbs/Gossiping/M.1495711757.A.9ED.html download to ptt/Gossiping/21180_4.txt, time: 3.990000
/bbs/Gossiping/M.1495711839.A.CA4.html download to ptt/Gossiping/21180_5.txt, time: 4.060000
/bbs/Gossiping/M.1495711895.A.FFB.html download to ptt/Gossiping/21180_6.txt, time: 4.830000
/bbs/Gossiping/M.1495711908.A.C90.html download to ptt/Gossiping/21180_7.txt, time: 4.490000
/bbs/Gossiping/M.1495711934.A.54F.html download to ptt/Gossiping/21180_8.txt, time: 8.400000
/bbs/Gossiping/M.1495712004.A.7D4.html download to ptt/Gossiping/21180_9.txt, time: 4.990000
/bbs/Gossiping/M.1495712029.A.606.html download to ptt/Gossiping/21180_10.txt, time: 4.650000
/bbs/Gossiping/M.1495712049.A.2F6.html download to ptt/Gossiping/21180_11.txt, time: 5.480000
/bbs/Gossiping/M.1495712055.A.5EC.html download to ptt/Gossiping/21180_12.txt, time: 3.730000
/bbs/Gossiping/M.1495712057.A.312.html download to ptt/Gossiping/21180_13.txt, time: 3.320000
/bbs/Gossiping/M.1495712062.A.A90.html download to ptt/Gossiping/21180_14.txt, time: 3.660000
/bbs/Gossiping/M.1495712090.A.36A.html download to ptt/Gossiping/21180_15.txt, time: 4.620000
/bbs/Gossiping/M.1495712116.A.001.html download to ptt/Gossiping/21180_16.txt, time: 3.750000
/bbs/Gossiping/M.1495712116.A.445.html download to ptt/Gossiping/21180_17.txt, time: 4.160000
/bbs/Gossiping/M.1495712149.A.7EC.html download to ptt/Gossiping/21180_18.txt, time: 4.360000
/bbs/Gossiping/M.1495712202.A.94D.html download to ptt/Gossiping/21180_19.txt, time: 3.330000
page 21179
/bbs/Gossiping/M.1495711181.A.7EC.html download to ptt/Gossiping/21179_0.txt, time: 4.000000
/bbs/Gossiping/M.1495711182.A.23D.html download to ptt/Gossiping/21179_1.txt, time: 4.540000
/bbs/Gossiping/M.1495711201.A.217.html download to ptt/Gossiping/21179_2.txt, time: 4.510000
/bbs/Gossiping/M.1495711277.A.773.html download to ptt/Gossiping/21179_3.txt, time: 4.740000
/bbs/Gossiping/M.1495711304.A.C50.html download to ptt/Gossiping/21179_4.txt, time: 4.870000
/bbs/Gossiping/M.1495711312.A.286.html download to ptt/Gossiping/21179_5.txt, time: 3.490000
/bbs/Gossiping/M.1495711351.A.CB5.html download to ptt/Gossiping/21179_6.txt, time: 4.950000
/bbs/Gossiping/M.1495711363.A.D51.html download to ptt/Gossiping/21179_7.txt, time: 5.530000
/bbs/Gossiping/M.1495711373.A.C4E.html download to ptt/Gossiping/21179_8.txt, time: 4.960000
/bbs/Gossiping/M.1495711423.A.6C5.html download to ptt/Gossiping/21179_9.txt, time: 5.590000
/bbs/Gossiping/M.1495711426.A.A9B.html download to ptt/Gossiping/21179_10.txt, time: 5.000000
/bbs/Gossiping/M.1495711445.A.B9B.html download to ptt/Gossiping/21179_11.txt, time: 6.210000
/bbs/Gossiping/M.1495711468.A.ECF.html download to ptt/Gossiping/21179_12.txt, time: 6.070000
/bbs/Gossiping/M.1495711468.A.5B0.html download to ptt/Gossiping/21179_13.txt, time: 5.390000
/bbs/Gossiping/M.1495711506.A.D55.html download to ptt/Gossiping/21179_14.txt, time: 6.280000
/bbs/Gossiping/M.1495711540.A.7B0.html download to ptt/Gossiping/21179_15.txt, time: 5.880000
/bbs/Gossiping/M.1495711581.A.7E6.html download to ptt/Gossiping/21179_16.txt, time: 8.480000
/bbs/Gossiping/M.1495711605.A.F59.html download to ptt/Gossiping/21179_17.txt, time: 6.040000
/bbs/Gossiping/M.1495711632.A.82C.html download to ptt/Gossiping/21179_18.txt, time: 4.770000
/bbs/Gossiping/M.1495711640.A.85B.html download to ptt/Gossiping/21179_19.txt, time: 12.090000
page 21178
/bbs/Gossiping/M.1495710538.A.1B8.html download to ptt/Gossiping/21178_0.txt, time: 6.430000
/bbs/Gossiping/M.1495710547.A.D6A.html download to ptt/Gossiping/21178_1.txt, time: 9.310000
/bbs/Gossiping/M.1495710563.A.5A7.html download to ptt/Gossiping/21178_2.txt, time: 4.950000
/bbs/Gossiping/M.1495710611.A.424.htmlTraceback (most recent call last):
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 386, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 382, in _make_request
    httplib_response = conn.getresponse()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/adapters.py", line 438, in send
    timeout=timeout
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 649, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/util/retry.py", line 357, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/packages/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 386, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 382, in _make_request
    httplib_response = conn.getresponse()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
requests.packages.urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./crawler.py", line 177, in <module>
    main(sys.argv[1:])
  File "./crawler.py", line 158, in main
    downloadForum(forumName, startPage, pages)
  File "./crawler.py", line 140, in downloadForum
    downloadPage(str(articleCount), forumName, pageNum, 'https://www.ptt.cc'+link)
  File "./crawler.py", line 82, in downloadPage
    pageText = getPageText(link)
  File "./crawler.py", line 13, in getPageText
    pageText=requests.get(link, cookies={'over18':'1'}).text
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/api.py", line 72, in get
    return request('get', url, params=params, **kwargs)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/sessions.py", line 518, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/sessions.py", line 639, in send
    r = adapter.send(request, **kwargs)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/adapters.py", line 488, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
